### PR TITLE
Feature fldyn euler constitutive law

### PR DESCRIPTION
**Pull request by rubenzorrilla**
This PR adds a new Euler constitutive law (no shear stress) as well as 4 new unit tests:
- 2D newtonian fluid test
- 3D newtonian fluid test
- 2D eulerian fluid test
- 3D eulerian fluid test

### CODE REVIEW

File: applications/FluidDynamicsApplication/tests/cpp_tests/test_fluid_dynamics_constitutive_laws.cpp [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+			p_elem_prop->SetValue(DYNAMIC_VISCOSITY, 3.0e-01);
+			p_elem_prop->SetValue(CONSTITUTIVE_LAW, p_cons_law);
+
+			// Element creation
+			model_part.CreateNewNode(1, 0.0, 0.0, 0.0);
+			model_part.CreateNewNode(2, 1.0, 0.0, 0.0);
+			model_part.CreateNewNode(3, 0.0, 1.0, 0.0);
+			std::vector<ModelPart::IndexType> elemNodes {1, 2, 3};
+			model_part.CreateNewElement("Element2D3N", 1, elemNodes, p_elem_prop);
+			Element::Pointer p_element = model_part.pGetElement(1);
```
Comment (jcotela) [inline applications/FluidDynamicsApplication/tests/cpp_tests/test_fluid_dynamics_constitutive_laws.cpp]: This block is repeated in all tests (and poorly indented). Can this be moved to a function?

**Review by jcotela — APPROVED** [review]
Yay! 825 lines of code to add zeros to a matrix!

**Comment by rubenzorrilla** [comment]
The most expensive zeros ever. :1st_place_medal:

File: applications/FluidDynamicsApplication/tests/cpp_tests/test_fluid_dynamics_constitutive_laws.cpp (hunk 1 above)
Comment (rubenzorrilla) [reply applications/FluidDynamicsApplication/tests/cpp_tests/test_fluid_dynamics_constitutive_laws.cpp]: I thought about that once I finished the test. It should be easy to do it.

**Pull request merged**